framework/src/org/checkerframework/framework/source/SourceChecker.java

```diff
@@ -175,18 +175,14 @@
     // org.checkerframework.framework.flow.CFAbstractTransfer.sequentialSemantics
     "concurrentSemantics",
 
-    // Whether to use conservative checks for unannotated bytecode; these are configured
-    // by the specific type checker using @DefaultInUncheckedCodeFor and @DefaultQualifierInHierarchyInUncheckedCode.
-    // They may require more annotations or stub files.
-    // "unsafeDefaultsForUnannotatedBytecode",
-    // TODO: temporary option to turn on sound behavior.
-    "safeDefaultsForUnannotatedBytecode",
-
-    // Whether to use conservative checks for unannotated source code; these are configured
-    // by the specific type checker using @DefaultInUncheckedCodeFor and @DefaultQualifierInHierarchyInUncheckedCode
-    // and only apply to source code that is not marked as @AnnotatedFor the checker
-    // that is being executed.
-    "useSafeDefaultsForUnannotatedSourceCode",
+    // Whether to use unchecked code defaults for bytecode and/or source code; these are configured
+    // by the specific type checker using @Default[QualifierInHierarchy]InUncheckedCode[For].
+    // This option takes an argument "source,bytecode" the default is
+    // -source,-bytecode (eventually this will be changed to -source,bytecode).
+    // Note, if unchecked code defaults are turned on for source code, the unchecked
+    // defaults are not applied to code in scope of an @AnnotatedFor.
+    // org.checkerframework.framework.source.SourceChecker.useUncheckedCodeDefault
```

Comment:
>What is this supposed to mean?

That's the  " pointer to one prominent use of the option".  This is the format that the other options use.